### DISCUSSION TITLE
Zeek notice.log

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#6020] [Unsupported Versions] [answered]
Hi,
Im trying to use zeek to alert when unknown traffic (outside of a baseline traffic I’ve created).
The problem is that the _**notice.log**_ under _**/nsm/zeek/logs/current**_ shows the following:

    “note”:”CaptureLoss::Too_Little_Traffic”,”msg”:”Only observed 0 TCP ACKs and was expecting at least 1.”,”peer_descr”:”worker-1-2”,”actions”:”[“Notice::ACTION_LOG”]
 
    “note”:”CaptureLoss::Too_Little_Traffic”,”msg”:”Only observed 0 TCP ACKs and was expecting at least 1.”,”peer_descr”:”worker-1-1”,”actions”:”[“Notice::ACTION_LOG”]

The monitoring interface does get traffic so I don't understand why it says _**Too_Little_Traffic**_
I even tried to use tcpreplay on the monitoring interface to make sure that traffic not flowing is not the issue.

Does anyone have any idea what could cause the CaptureLoss message?

**Comment by dougburks** [comment, marked as answer]
How did you configure the system?

Is the system connected to a tap or span port?

How did you verify that the monitoring interface is receiving traffic?

How much traffic is it receiving?

**Reply by yula-21** [reply]
- The system has 2 network adapters, one for management and the other for monitoring.
- The monitoring interface is supposed to connect to a tap but the system is currently in development so the setup is a bit different  - to ensure traffic I run tcpreplay with a large pcap file but I don’t know to say how much traffic flows, I do know that there aren't 0 as the log says.

Are there tests I can run to check where the problem is coming from?
I’ve tried to read a bit about notice logs but haven't come across something similar to my problem.

**Reply by yula-21** [reply]
Another thing I noticed is that **_conn.log_** is getting logs from the traffic flowing in the monitoring interface.

I dont fully understand the architecture of zeek, is it possible that conn logs get the traffic but _**notice.log**_ is showing _“0 TCP ACKs”_?

**Reply by dougburks** [reply]
If you are just running a periodic tcpreplay or if there are gaps in the pcap file, then it may be entirely possible that there are windows of time where no traffic hits the sniffing interface. If that's the case, then the Zeek notice is correct and nothing to be concerned about.

**Reply by yula-21** [reply]
Thank you for your reply, I checked and what you’re saying is true.
The only problem I am still seeing is that the traffic flowing to zeek seems to skip certain ip addresses, I am getting traffic from a single subnet.
Is there a configuration file where it could be set to get traffic from certain subnets?

**Reply by dougburks** [reply]
You could check and see if there is a BPF defined:
https://docs.securityonion.net/en/2.3/bpf.html

If you have further questions or problems, please start a separate discussion rather than replying to this Zeek notice.log discussion which is now marked as answered.

**Reply by gdianq** [reply]
> If you are just running a periodic tcpreplay or if there are gaps in the pcap file, then it may be entirely possible that there are windows of time where no traffic hits the sniffing interface. If that's the case, then the Zeek notice is correct and nothing to be concerned about.

I am new to zeek and I have encountered this problem recently. How to solve this problem?

**Reply by dougburks** [reply]
> If you are just running a periodic tcpreplay or if there are gaps in the pcap file, then it may be entirely possible that there are windows of time where no traffic hits the sniffing interface. If that's the case, then the Zeek notice is correct and nothing to be concerned about.

@gdianq If you are asking about this snippet that you copied, then it is not a problem and that's explained in the second sentence. If you have other questions or problems, please start a new discussion rather than replying to somebody else's discussion.
